jvm/src/main/kotlin/app/cash/trifle/TrifleErrors.kt

```diff
@@ -0,0 +1,7 @@
+package app.cash.trifle
+
+class NoTrustAnchorException(message: String? = null, cause: Throwable? = null) : Exception(message, cause)
```

Comment:
if you'd like to continue pursue an exception handling route, I'd recommend that this checked exception be defined under a sealed class.